_data/campaigns.json

```diff
@@ -0,0 +1,22 @@
+[
+    {
+        "photo": "no-photo.jpg",
+        "_id": "603190d3a08aa76a99382abd",
+        "title": "iPhonesp",
```

Comment:
we will see that a lot while testing, we better have it without the typo :D

Reply:
resolved